pyserini/analysis/pyanalysis.py

```diff
@@ -0,0 +1,131 @@
+from ..pyclass import JArabicAnalyzer
+from ..pyclass import JBengaliAnalyzer
+from ..pyclass import JCJKAnalyzer
+from ..pyclass import JEnglishStemmingAnalyzer
+from ..pyclass import JFreebaseAnalyzer
+from ..pyclass import JFrenchAnalyzer
+from ..pyclass import JGermanAnalyzer
+from ..pyclass import JHindiAnalyzer
+from ..pyclass import JSpanishAnalyzer
+from ..pyclass import JTokenizeOnlyAnalyzer
+from ..pyclass import JTweetAnalyzer
+
+
+def get_default_analyzer():
```

Comment:
What about `get_analyzer()` as a function that takes a enum to specify which one? And if called with zero args, returns the default?

Reply:
Sure will implement